<?xml version="1.0"?>
<launch>
    <param name="WORLD_NAME" value="$(env WORLD_NAME)" />
    <param name="MODEL_S3_BUCKET" value="$(env MODEL_S3_BUCKET)" />
    <param name="MODEL_S3_PREFIX" value="$(env MODEL_S3_PREFIX)" />
    <param name="TRAINING_JOB_ARN" value="$(env TRAINING_JOB_ARN)" />
    <param name="AWS_REGION" value="$(env APP_REGION)" />

    <param name="JOB_TYPE" value="TRAINING" />

    <include file="$(find deepracer_simulation_environment)/launch/racetrack_with_racecar.launch">
        <arg name="world_name" value="$(env WORLD_NAME)" />
        <arg name="kinesis_video_stream_name" value="$(env KINESIS_VIDEO_STREAM_NAME)" />
    </include>

    <node name="agent" pkg="deepracer_simulation_environment" type="run_local_rl_agent.sh" output="screen" required="true"/>
</launch>
